src/components/DictionaryList/DictionaryListPresentation.js

```diff
@@ -152,6 +152,7 @@ function DictionaryListPresentation({
                 </thead>
                 <tbody className="bg-white divide-y divide-gray-300">
                   {items.pages.map((page, index) => (
+                    // eslint-disable-next-line react/no-array-index-key
```

Comment:
The aim with these new eslint rules is to gradually fix old code as we touch it. If you could please fix this error rather than disabling the eslint that would be great. Thanks.

Reply:
There's not uid for these elements.  I'm not sure how to change it from an index when no uid is present.